lib/taskana-core/src/main/java/pro/taskana/monitor/api/reports/TimeIntervalReportBuilder.java

```diff
@@ -26,69 +26,69 @@
     extends Report.Builder<I, H> {
 
   /**
-   * Adds a list {@linkplain TimeIntervalColumnHeader}s to the builder to subdivide the report into
-   * clusters.
+   * Adds a list {@linkplain TimeIntervalColumnHeader}s to the Builder to subdivide the {@link
+   * Report} into clusters.
    *
-   * @param columnHeaders the column headers the report should consist of.
+   * @param columnHeaders the column headers the {@link Report} should consist of.
    * @return the TimeIntervalReportBuilder
    */
   B withColumnHeaders(List<H> columnHeaders);
 
   /**
-   * If this filter is used, the days of the report are counted in working days.
+   * If this filter is used, the days of the Report are counted in working days.
    *
    * @return the TimeIntervalReportBuilder
    */
   B inWorkingDays();
 
   /**
-   * Adds a list of workbasket ids to the builder. The created report contains only tasks with a
-   * workbasket id in this list.
+   * Adds a list of workbasket ids to the builder. The created {@link Report} contains only {@link
+   * Task}s with a workbasketId in this list.
    *
    * @param workbasketIds a list of workbasket ids
    * @return the TimeIntervalReportBuilder
    */
   B workbasketIdIn(List<String> workbasketIds);
 
   /**
-   * Adds a list of states to the builder. The created report contains only tasks with a state in
-   * this list.
+   * Adds a list of states to the builder. The created {@link Report} contains only {@link Task}s
+   * with a state in this list.
    *
    * @param states a list of states
    * @return the TimeIntervalReportBuilder
    */
   B stateIn(List<TaskState> states);
 
   /**
-   * Adds a list of classificationCategories to the builder. The created report contains only tasks
-   * with a category in this list.
+   * Adds a list of classificationCategories to the builder. The created {@link Report} contains
+   * only {@link Task}s with a category in this list.
```

Comment:
In other classes you used {@linkplan Task Tasks} instead of "{@link Task}s". Which one is correct?